### DISCUSSION TITLE
[BUG] Issue with Displaying cover Image on Blog Post

### DISCUSSION THREAD

**Discussion by cubxxw** [adityatelange/hugo-PaperMod#1375] [Q&A] [answered]
Hello,

I've encountered an issue with displaying the `cover` image in one of my blog posts. The post in question is titled "2023 Annual Summary Reflections and Aspirations," and it's published on my blog here: [2023 Annual Summary Reflections and Aspirations](https://nsddd.top/posts/2023-annual-summary-reflections-and-aspirations/).

In the markdown file for this post ([source code](https://github.com/cubxxw/blog/blob/main/content/en/posts/2023-annual-summary-reflections-and-aspirations.md)), I have specified a cover image as follows:

```yaml
cover:
    image: images/blog/sm202309161719007.png
    caption: 'about 2023 year'
```

However, the image does not appear on the blog post as expected. This seems to be an issue with rendering or path resolution. Could you please assist in resolving this matter? Any guidance on how to fix this issue would be greatly appreciated.

Thank you for your time and assistance.

**Comment by adityatelange** [comment, marked as answer]
Can you share image path in source? sm202309161719007.png
![image](https://github.com/adityatelange/hugo-PaperMod/assets/21258296/249b1628-ffb2-40e1-a8ab-4335c79c86ce)

**Comment by yuhanwww** [comment]
You can try `hidden:false`under the caption line and see if it appears. It could be that in your config file, you have set `hidden:true` that will not display your cover photo by default.
